components/chat_widget/src/components/ocs-chat/ocs-chat.css

```diff
@@ -84,3 +84,105 @@ textarea {
 .overflow-y-auto::-webkit-scrollbar-thumb:hover {
   @apply bg-gray-400;
 }
+
+/* Markdown content styling for chat messages */
```

Comment:
I think most of these styles could be replaced by using the [tailwind typography plugin](https://github.com/tailwindlabs/tailwindcss-typography). Here's an example recipe from the Astro docs: https://docs.astro.build/en/recipes/tailwind-rendered-markdown/